evm/verify-contract.js

```diff
@@ -297,14 +346,24 @@ async function main(options) {
 if (require.main === module) {
     const program = new Command();
 
-    program.name('balances').description('Display balance of the wallet on specified chains.');
+    program.name('verify-contract').description('Verify selected contracts on specified chains.');
 
     addBaseOptions(program, { ignorePrivateKey: true, address: true });
 
     program.addOption(new Option('-c, --contractName <contractName>', 'contract name'));
     program.addOption(new Option('-d, --dir <dir>', 'contract artifacts dir'));
     program.addOption(new Option('--args <args>', 'contract args'));
     program.addOption(new Option('--constructorArgs <constructorArgs>', 'contract constructor args'));
+    program.addOption(new Option('--deployer <deployer>', 'interchain token deployer address'));
+    program.addOption(new Option('--salt <salt>', 'interchain token deployment salt'));
+    program.addOption(new Option('--rawSalt <rawSalt>', 'raw deployment salt').env('RAW_SALT'));
+    program.addOption(new Option('--minter <minter>', 'interchain token minter address'));
+    program.addOption(new Option('--tokenId <tokenId>', 'interchain token tokenId'));
+    program.addOption(
+        new Option('--implementationType <implementationType>', 'token manager implementation type').choices(['0', '1', '2', '3']),
+    );
+    program.addOption(new Option('--tokenManagerAddress <tokenManagerAddress>', 'token manager proxy address'));
+    program.addOption(new Option('--params <params>', 'token manager deployment params'));
```

Comment:
too many options, just need tokenId, and minter if it was set, and everything else should be calculated dynamically. we just need to do this once on most chains anyways